tapdb/sqlc/queries/metadata.sql

```diff
@@ -0,0 +1,2 @@
+-- name: AssetsDBSize :one
+SELECT pg_catalog.pg_database_size(current_database()) AS size;
```

Comment:
We'll need a solution that also works w/ sqlite. 

Got this from Claude (untested): 
```
SELECT page_count * page_size AS size_in_bytes 
FROM pragma_page_count(), pragma_page_size();
```

Reply:
the above recommendation from Claude seems to be working fine